hmac/core/src/main/scala/com/gu/pandahmac/HMACClient.scala

```diff
@@ -0,0 +1,21 @@
+package com.gu.pandahmac
+
+import com.gu.hmac.HMACHeaders
+
+import java.net.URI
+
+/**
```

Comment:
This comment might not be true over time, and IIUC as a result of this release should no longer be true soon-ish: hostage to fortune, or pragmatically helpful in what is usually a slow-moving domain?

Reply:
This particular class actually isn't used anywhere yet, so I guess it's already misleading! https://github.com/guardian/flexible-content/pull/5296 will start consuming it, and then it makes sense to role it out to `workflow`, `typerighter`, etc.

I think you're right that it's the kind of comment that can go out-of-date over time, I'll delete it...